docs/composer/configuration.md

```diff
@@ -1,115 +1,104 @@
-# Configuration
+---
+title: Configuration Overview
+label: Platformatic Composer Configuration
+---
 
-Platformatic Composer configured with a configuration file. It supports the use
-of environment variables as setting values with [environment variable placeholders](#environment-variable-placeholders).
+# Configuration of Platformatic Composer
 
-## Configuration file
+Platformatic Composer can be configured with a [configuration file](#configuration-files) in the different file formats below. The Composer also supports use of environment variables as setting values with [environment variable placeholders](https://docs.platformatic.dev/docs/composer/configuration#environment-variable-placeholders). 
 
-If the Platformatic CLI finds a file in the current working directory matching
-one of these filenames, it will automatically load it:
+## Configuration Files
 
-- `platformatic.composer.json`
-- `platformatic.composer.json5`
-- `platformatic.composer.yml` or `platformatic.composer.yaml`
-- `platformatic.composer.tml` or `platformatic.composer.toml`
+Platformatic Composer will automatically detect and load configuration files found in the current working directory. The supported file names include:
 
-Alternatively, a [`--config` option](/reference/cli.md#composer) with a configuration
-filepath can be passed to most `platformatic composer` CLI commands.
+- `platformatic.json`
+- `platformatic.json5`
+- `platformatic.yml` or `platformatic.yaml`
+- `platformatic.tml` or `platformatic.toml`
 
-The configuration examples in this reference use JSON.
+To specify a configuration file manually, use the [`--config` option](../cli.md#composer) with any `platformatic composer` CLI command. 
 
-### Supported formats
+### Supported File Formats
 
-| Format | Extensions |
-| :-- | :-- |
-| JSON | `.json` |
-| JSON5 | `.json5` |
-| YAML | `.yml`, `.yaml` |
-| TOML | `.tml` |
+Platformatic Composer supports the following file formats, their file extensions and the ability to include comments:
 
-Comments are supported by the JSON5, YAML and TOML file formats.
+| Format | Extensions         | Comments Supported |
+|--------|--------------------|--------------------|
+| JSON   | `.json`            | No                 |
+| JSON5  | `.json5`           | Yes                |
+| YAML   | `.yml`, `.yaml`    | Yes                |
+| TOML   | `.tml`, `.toml`    | Yes                |
```

Comment:
Can we abstract those sections about the file formats and extensions in its own page?

Reply:
Done.